src/ex4/server.js

```diff
@@ -1 +1,25 @@
-// Express boilerplate, hosting the `dist` file, connecting to the routes
\ No newline at end of file
+// Express boilerplate, hosting the `dist` file, connecting to the routes
+// make bluebird default Promise
+
+const express = require("express");
+const app = express();
+const tasks = require("./server/routes/api.js");
+const logger = require("./server/middleware/logger.js");
+const cors = require("cors");
+const clearCache = require("./server/services/clearCache.js");
+const cron = require("node-cron");
+
+app.use(express.json());
+app.use(cors());
+app.use(logger);
+app.use("/tasks", tasks);
+
```

Comment:
you need to serve your dist folder from here using express.static.
it's probably the reason why you had loading issues.